EIPS/eip-flash-loans.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: <to be assigned>
+title: Flash Loans
+author: Alberto Cuesta Cañada (@albertocuestacanada), Fiona Kobayashi (@fifikobayashi), fubuloubu (@fubuloubu)
+discussions-to: https://ethereum-magicians.org/t/flash-loan-eip-early-draft/4993/2
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-11-15
+---
+
+<!--This is the suggested template for new EIPs.
+
+Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`.
+
+The title should be 44 characters or less.-->
+
+## Simple Summary
+
+Flash loans allow smart contracts to lend an amount of tokens without a requirement for collateral, with the condition that they must be returned within the same transaction.
+
+The aim of this ERC is to provide a standard interfaces and processes for flash lenders and borrowers, allowing for flash loan integration without a need to consider each particular implementation.
+
+## Abstract
```

Comment:
The abstract should be a technical overview of the EIP. I don't recommend it for an ERC.

Reply:
Please check again, thanks.